solution/ui/regulations/eregs-vite/src/views/Resources.vue

```diff
@@ -729,7 +731,6 @@ export default {
         if (this.queryParams.q) {
             this.searchQuery = this.queryParams.q;
             this.synonyms = await this.retrieveSynonyms(this.queryParams.q)
-            this.multiWordQuery()
```

Comment:
this is a computed prop, not a method.  It should update reactively as its dependency (`this.searchQuery`) changes.